google/cloud/spanner/instance_admin_client.h

```diff
@@ -55,8 +55,8 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
  * This class uses `StatusOr<T>` to report errors. When an operation fails to
```

Comment:
Note to self:  Perhaps we should trying to suppress the generated documentation for this deprecated class, or at least have the documentation so marked.

Reply:
Done